conf/core/ConfigOptions.asciidoc

```diff
@@ -221,6 +221,16 @@ Date format string used by the building date tag value.  See QDateTime::fromStri
 
 Tag key used by the building.review.if.secondary.newer configuration option.
 
```

Comment:
This new option really doesn't have anything to do with this merging task, since it deals with auto-merging during conflation and not manual merging.  Someone had requested it, so I just went ahead and did with the other merging changes.